api/v1/services/StorageService.js

```diff
@@ -183,7 +183,7 @@ module.exports.persistUpload = function (upload, file) {
 };
 
 /**
- * Retrieves an upload from remote storage
+ * Retrieves an upload from _remote storage
```

Comment:
Please don't find-and-replace things that aren't related to the feature you're working on. This doesn't make sense